src/executor/paginator.rs

```diff
@@ -80,10 +80,22 @@ where
     /// Get the total number of pages
     pub async fn num_pages(&self) -> Result<usize, DbErr> {
         let num_items = self.num_items().await?;
-        let num_pages = (num_items / self.page_size) + (num_items % self.page_size > 0) as usize;
+        let num_pages = self.compute_pages_number(num_items);
         Ok(num_pages)
     }
 
+    /// Get the total number of items and pages in a tuple as (number_of_items, number_of_pages)
+    pub async fn num_items_and_pages(&self) -> Result<(usize, usize), DbErr> {
```

Comment:
Hey @fistons, thanks for the contributions! I'm just thinking would it be better to return a struct with named fields: `number_of_items` and `number_of_pages`?

Reply:
Yes, sure things!